HLTrigger/Configuration/python/HLT_75e33/modules/hgcalLayerClustersHSciL1Seeded_cfi.py

```diff
@@ -0,0 +1,9 @@
+import FWCore.ParameterSet.Config as cms
+
+import hgcalLayerClustersEEL1Seeded
```

Comment:
The suggestion is to remove this import, and write the explicit version of `hgcalLayerClustersHSciL1Seeded` (the one returned by `hgcalLayerClustersHSciL1Seeded.dumpPython()`). The idea is that every `cfi` is standalone, it does not import other modules (the exception would be importing the default `cfi` created by `fillDescriptions`, but this is the case here).

Reply:
fixed, thank you